client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -0,0 +1,465 @@
+import styled from '@emotion/styled';
+import { translate, useTranslate } from 'i18n-calypso';
+import { ChangeEvent, useCallback, useState } from 'react';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormInput from 'calypso/components/forms/form-text-input';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { ConfirmationModal } from '../confirmation-modal';
+import SyncOptionsPanel, { CheckboxOptionItem } from '../sync-options-panel';
+import { useCheckSyncStatus } from '../use-site-sync-status';
+import { StagingSiteSyncLoadingBarCardContent } from './staging-site-sync-loading-bar-card-content';
+
+const synchronizationOptions: CheckboxOptionItem[] = [
+	{
+		name: 'sqls',
+		label: 'Site Database (SQL)',
+		subTitle: translate(
+			'Overwrite the database, including any posts, pages, products, or orders with staging data.'
```

Comment:
As we use this text in both contexts, let's unify it:

```suggestion
			'Overwrite the database, including any posts, pages, products, or orders.'
```